src/audio/pipeline.c

```diff
@@ -798,10 +798,11 @@ void pipeline_schedule_cancel(struct pipeline *p)
 					  "failed, err = %d", err);
 }
 
-static void pipeline_task(void *arg)
+static uint64_t pipeline_task(void *arg, uint64_t udelay)
 {
 	struct pipeline *p = arg;
 	int err;
+	int ret = 0;
 
```

Comment:
Same here. No need for ret. Just return 0 or error where appropriate.

Reply:
Updated